internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -94,7 +94,7 @@ var (
 				Tooling:   &ContainerImageTag{Name: "skuba-tooling", Tag: "0.1.0"},
 			},
 			AddonsVersion: AddonsVersion{
-				Cilium:        &AddonVersion{"1.5.3", 2},
+				Cilium:        &AddonVersion{"1.6.6", 3},
```

Comment:
There is already a 1.18 k8s version that is wrongly pointing at 1.5.3. That should be changed